### DISCUSSION TITLE
I am trying to create a Model purely with code, but the loaded Texture is not displaying correctly.

### DISCUSSION THREAD

**Discussion by jkh404** [stride3d/stride#2588] [Newcomers]
I am learning Stride3D and Stride.CommunityToolkit, but I am stuck on this issue. I’ve tried hard to solve it, but it’s still not working. The screenshot code is mimicking the Example05_ProceduralGeometry sample project from Stride.CommunityToolkit.

I have assigned the correct value to the Texture field of ComputeTextureColor, but the image is not displayed on the square. Can someone tell me what the correct approach is? Thanks in advance.

![image](https://github.com/user-attachments/assets/f1e6f505-2ddb-4555-a254-0f121b83bb77)

**Comment by VaclavElias** [comment]
Is this example working for you? https://stride3d.github.io/stride-community-toolkit/tutorials/textures/add-a-texture.html

I have been putting these examples together but some scenarios are also unknown for me. I believe, there needs to be done somewhere some small update in your example to work.

**Reply by jkh404** [reply]
Thank you for your answer. I should be able to find the solution within the Create3DPrimitive method. I’ll give it a try.
